static/app/components/events/eventReplay.tsx

```diff
@@ -51,4 +46,12 @@ export default function EventReplay({replayId, orgSlug, projectSlug}: Props) {
 const PlayerContainer = styled('div')`
   max-width: 420px;
   margin-top: ${space(2)};
+
+  background: ${p => p.theme.background};
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  overflow: hidden;
+  gap: ${space(1)};
```

Comment:
This CSS seems to be repeated a bit, wonder if it's worth making as its own [styled] component?

Reply:
it is! it's used 4 times in `styled('div')...` statements.

I was able to put the shared css into a new component `FluidHeight` which all 4 of those examples.